OpenRA.Mods.Common/Scripting/Properties/AircraftProperties.cs

```diff
@@ -16,12 +16,13 @@
 
 namespace OpenRA.Mods.Common.Scripting
 {
+	[ExposedForAIScript]
```

Comment:
When I sugggested this I didn't forsee the design compromises it would entail.  This is still a cleaner way to *declare* the interfaces, but the current set of tradeoffs aren't worth it.

Of course, the best case would be if you could solve the things I commented about below without having to add more complexity and while still keeping these attributes at the method level.  This is probably possible by doing the class level filtering I mentioned below on the methods/properties instead.